pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -142,7 +143,7 @@ def on_epoch_end(self, epoch, logs=None):
 
     def on_train_end(self, logs=None):
         if self.stopped_epoch > 0 and self.verbose > 0:
-            logging.info('Epoch %05d: early stopping' % (self.stopped_epoch + 1))
+            logging.info(f'Epoch {self.stopped_epoch + 1:05d}: early stopping')
```

Comment:
not directly to this line, but we shall be sure that all this new logging is tested (covered by tests)

Reply:
This did occur to me, but I didn't know how to test this logging, any idea on this?

Reply:
I usually tested that the message does not crash the code...